### PR TITLE
Added get listing endpoint

### PR DESCRIPTION
**Pull request by patrickbussmann**

### CODE REVIEW

File: resources/service.php [hunk 1]
```diff
@@ -348,6 +348,24 @@
...
+				    'location' => 'uri',
+				    'required' => true
+			    ],
+			    'curr_abbr' => [
+				    'type' => 'string',
+				    'location' => 'query',
+				    'required' => false
+			    ]
+		    ]
+	    ],
```
Comment (ricbra) [inline resources/service.php:368]: Could you apply the same indenting as the rest of the file?

File: tests/Discogs/Test/ClientTest.php [hunk 2]
```diff
@@ -253,6 +253,22 @@ public function testChangeOrder()
...
+			'curr_abbr' => 'USD'
+		]);
+
+		$this->assertFalse($response['allow_offers']);
+		$this->assertSame('For Sale', $response['status']);
+		$this->assertArrayHasKey('original_price', $response);
+		$this->assertSame('https://api.discogs.com/marketplace/listings/1?curr_abbr=USD', $history->getLastRequest()->getUrl());
+		$this->assertSame('GET', $history->getLastRequest()->getMethod());
+	}
+
```
Comment (ricbra) [inline tests/Discogs/Test/ClientTest.php:271]: Could you fix the indenting? (spaces instead of tabs)

**Comment by ricbra** [comment]
Thanks @patrickbussmann

**Comment by patrickbussmann** [comment]
@ricbra I updated this. Sorry 👍 
Different settings 😅

**Comment by patrickbussmann** [comment]
@ricbra? 😇

**Comment by ricbra** [comment]
🎉

**Pull request merged**